competition/track2/train/train.py

```diff
@@ -88,7 +88,7 @@ def train(input_path, output_path):
             print(f"Adding data for vehicle id {id} in scenario {scenario}.")
 
             with open(
-                scenario_path / (f"Agent-history-vehicle-{id}.pkl"),
+                scenario_path / (f"sumo_sumo_1_Agent-history-vehicle-{id}.pkl"),
```

Comment:
If you hard code "sumo_sumo_*" string, what happens when a differently named hidden dataset is used during docker-based re-training?